checkfiles.py

```diff
@@ -323,11 +323,11 @@ def validate_crispr(job, filePath):
 
             except subprocess.CalledProcessError as e:
                 errors['CRISPR_PAM_info_extraction'] = 'Failed to extract information from ' + \
-                                                            local_path
+                                                            filePath
```

Comment:
Can we change `filePath` to `file_path`

Reply:
I would prefer to touch this code as little as possible, you never know what might call this function and you never know if they use positional or keyword argument style.

Reply:
Oh ok!